### PR TITLE
feat(config): add config for Minoston MP21ZD

### PR DESCRIPTION
**Pull request by 3van**
Parameter definitions based off of the manual linked in metadata, and tested locally with the one that just arrived at my house.

### CODE REVIEW

File: packages/config/config/devices/0x0312/mp21zd.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Last state",
+					"value": 2
+				}
+			]
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21zd.json]: We now have [templates](https://github.com/zwave-js/node-zwave-js/blob/6c4d9d026a44d656e3b4d1011d7b1215cfa29ce3/packages/config/config/devices/templates/master_template.json#L256-L405) for the variants of this config option. Please choose the matching one and use it.

File: packages/config/config/devices/0x0312/mp21zd.json [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 4,
+			"unsigned": true
+		},
+		"11": {
+			"label": "Minimum Brightness",
+			"valueSize": 1,
+			"unit": "%",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21zd.json]:
```suggestion
```
Nitpicking,  but brightness in Z-Wave is not in %. See the note above https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=readwrite-only

File: packages/config/config/devices/0x0312/mp21zd.json [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+		"13": {
+			"label": "Temperature Report Interval",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 60,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		"14": {
+			"label": "Temperature Delta Threshold For Report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21zd.json]:
```suggestion
			"label": "Temperature Report Change Threshold",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks - a few minor edits please

**Comment by AlCalzone** [comment]
Hey @3van are you planning on finishing this PR?

**Review by AlCalzone — APPROVED** [review]

**Comment by geeregman** [comment]
First, thank you for all your work.
I just want to comment; I found that the MP21ZD configuration values are 'inverted':
I needed the switch to be in the on state after return from a power outage.
I had to set the 'state after power failure' to OFF in order for the switch to be ON after power is returned, which is backwards from what I want. Also, the LED indicator setting is backwards, that is, setting LED on with load OFF, in actuality the load is ON when the LED is ON.
I am using homeassistant and ZWavejs2mqtt control panel.  Everything works as it should, just that the configuration settings seem to be inverted................

**Comment by AlCalzone** [comment]
@geeregman any chance you have a manual to confirm this against?

**Comment by geeregman** [comment]
Attached is the manual I have from Minoston.

In the 'Restores after power failure' section.
Value 1 is always OFF, and Value 2 is always ON.

In the 'LED Indicator section' the values in the guide are incorrect.

I have been in contact with Minoston regarding the incorrect LED indicator section.
attached is a copy an email from them.


thanks,
Greg

________________________________

**Comment by AlCalzone** [comment]
I don't see anything attached. Note that replying to issues via email does not preserve the attachments. You'll have to do that via the web.

**Comment by geeregman** [comment]
Here are the files (I think)
[ZwavesmartplugMinostonMP21ZP.pdf](https://github.com/zwave-js/node-zwave-js/files/7692910/ZwavesmartplugMinostonMP21ZP.pdf)
[MinostonEmailReply.pdf](https://github.com/zwave-js/node-zwave-js/files/7692924/MinostonEmailReply.pdf)

**Comment by AlCalzone** [comment]
Thanks, PR #3878 opened

**Pull request merged**